File: Java/5-String-based-Programs/60-toggle-each-character-in-a-string.java
import java.util.Scanner;

class Main {
    public static void main(String[] args) {
        Scanner getInput = new Scanner(System.in);
        
        System.out.println("Enter a String: ");
        String text = getInput.nextLine();

        String result = "";
        for (char ch : text.toCharArray()){
            if (Character.isLowerCase(ch)){
                result += Character.toUpperCase(ch);
            }
            else if (Character.isUpperCase(ch)){
                result += Character.toLowerCase(ch);
            }
            else{
                result += ch;
            }
        }

        System.out.println("The Toggled string is: " + result);
        getInput.close();
    }
}

File: Python/5-String-based-Programs/60-toggle-each-character-in-a-string.py
text = input("Enter a String: ")
result = ""

for ch in text:
    if ch.isupper():
        result += ch.lower()
    elif ch.islower():
        result+= ch.upper()
    else:
        result += ch

print("The Toggled string is:", result)

# Built-in Method
# text.swapcase()